After upgrade to typescript 2.6.1 will get IFindOptions error

After upgrade to typescript 2.6.1 will get this error.

`error TS2345: Argument of type ... is not assignable to parameter of type 'IFindOptions'.   Types of property 'where' are incompatible.`

Workaround: lock the typescript version to 2.5.3

Not sure if the issue similar to #106

@RobinBuschmann 

Typescript 
````
CalendarEvent.findOne<CalendarEvent>({
  where: {
    ce_id: req.params.ce_id,
    ce_created_by: req.user.user_id,
    ce_active: true,
    ce_deleted: false
  }
})
````

Will returns this error
````
error TS2345: Argument of type 
'{
    where: {
       ce_id: any;
       ce_created_by: any; ce_active: boolean; ce_deleted: boolean;
    };
}' 
is not assignable to parameter of type 'IFindOptions'.
Types of property 'where' are incompatible.
````

--- a/package.json
+++ b/package.json
@@ -45,7 +45,7 @@
     "@types/bluebird": "3.5.18",
     "@types/node": "6.0.41",
     "@types/reflect-metadata": "0.0.4",
-    "@types/sequelize": "4.0.73",
+    "@types/sequelize": "4.0.79",
     "es6-shim": "0.35.3"
   },
   "devDependencies": {

